app/routes/events/components/EventList.css

```diff
@@ -2,13 +2,15 @@
 
 .root {
   composes: contentContainer from '~app/styles/utilities.css';
+  margin-top: 12px;
 }
 
 .heading {
-  font-size: 20px;
-  border-bottom: 1px solid var(--border-gray);
-  padding-bottom: 10px;
-  margin-bottom: 20px;
+  font-size: 18px;
+  border-bottom: 2px solid var(--border-gray);
+  font-weight: 500;
```

Comment:
The added `font-weight` shouldn't be necessary since `.heading` is used on an `h2` tag.